src/api/public/apidocs-new/paths/source_project_name_cmd_release.yaml

```diff
@@ -0,0 +1,23 @@
+post:
+  summary: Release the project
+  description: |
+    Release source and binaries of the project, if you have the permissions to do so.
+  security:
+    - basic_authentication: []
+  parameters:
```

Comment:
These parameters should also be documented: `:targetproject, :targetreposiory, :repository, :setrelease`. See: https://github.com/openSUSE/open-build-service/blob/12bad1ae9bd785f9164f42c4bb8266d12fabd54e/src/api/app/controllers/source_controller.rb#L559

Reply:
*project* is already documented, we use *project_name* in the documentation. It's not an issue that we use *project_name* instead of *project* since it's a path parameter.

As for *user*, this should not be documented as even if someone passes a user in the query string, it's overwritten with the logged-in user:
https://github.com/openSUSE/open-build-service/blob/12bad1ae9bd785f9164f42c4bb8266d12fabd54e/src/api/app/controllers/source_controller.rb#L539-L541

Reply:
This should be addressed now. Please have another look.